docs/configuration.html

```diff
@@ -137,6 +151,8 @@ <h5>Updating Default Topic Configuration</h5>
     <li><code>log.message.timestamp.difference.max.ms</code></li>
   </ul>
 
+  From Kafka version 2.0.0 onwards, unclean leader election is automatically enabled by the controller when the config
```

Comment:
@hachikuji The changes to the controller for dynamic broker config update was made under https://issues.apache.org/jira/browse/KAFKA-6526 (already merged). So I thought it will be good to update the broker config doc for default topic configuration. @omkreddy is updating the controller for per-topic-override of `unclean.leader.election.enable` and I think his patch includes changes to the upgrade docs.

Reply:
Ah, ok. I missed the difference between KAFKA-6835 and KAFKA-6526.